lib/index.js

```diff
@@ -17,6 +19,7 @@ var get = require('obj-case');
 var TwitterAds = module.exports = integration('Twitter Ads')
   .option('page', '')
   .option('universalTagPixelId', '')
+  .option('identifier', 'productId')
```

Comment:
Doesn't look like this setting is in metadata yet. When you add it, could you make sure to have it validated as an enum to avoid javascript errors.